build.sbt

```
@@ -62,3 +72,16 @@ lazy val allProjects: Seq[ProjectReference] = Seq(
   `memeid4s-http4s`,
   `memeid4s-scalacheck`
 )
+
+lazy val micrositeSettings: Seq[Def.Setting[_]] = Seq(
+  micrositeName := "memeid",
+  micrositeDescription := "A JVM library for generating RFC-compliant Universal Unique Identifiers (UUIDs)",
+  micrositeBaseUrl := "memeid",
+  micrositeDocumentationUrl := "docs",
+  micrositeGithubOwner := "47degrees",
+  micrositeGithubRepo := "memeid",
+  micrositeOrganizationHomepage := "https://www.47deg.com",
+  micrositePushSiteWith := GitHub4s,
+  micrositeHighlightLanguages ++= Seq("java", "scala", "kotlin"),
```

I would remove `kotlin` until we have an integration for it

```suggestion
  micrositeHighlightLanguages ++= Seq("java", "scala"),
```